tox.ini

```diff
@@ -16,7 +16,7 @@ commands=
 
 [testenv:tests]
 commands=
-    pytest -v --doctest-modules iotlabsshcli --cov=iotlabsshcli
+    pytest --doctest-modules iotlabsshcli --cov=iotlabsshcli
```

Comment:
not a good reason sorry. You can't change things like this. This is totally unrelated to this PR !